panel/tests/ui/widgets/test_tabulator.py

```diff
@@ -1041,10 +1041,10 @@ def test_tabulator_frozen_rows(page, port):
     expected_text = """
     index
     col
-    8
-    Y
     1
     X
+    8
+    Y
```

Comment:
Why is this changed?

Reply:
I've decided to change the behavior, a) because it's much simpler and b) frozen rows should respect the original row order. While I could see an argument that the order in which the frozen row were defined should determine their ordering at the top of the table the actual implementation only did that correctly in the simplest case. So I've decided to switch to a consistent and simpler implementation.

Reply:
Will record this change in the CHANGELOG.